modules/wolfgame.py

```diff
@@ -471,11 +476,17 @@ def on_whoreply(cli, server, dunno, chan, dunno1,
         if not var.PINGING: return
         if user in (botconfig.NICK, nick): return  # Don't ping self.
 
-        if ('G' not in status and '+' not in status and not
-            is_user_stasised(user)[0] and not is_user_away(user)):
-
-            TO_PING.append(user)
-
+        if ('G' not in status and '+' not in status and not is_user_stasised(user)[0]):
```

Comment:
Per brrr's comment, this should check to see if the user actually joined, not if the user has voice, for two reasons:
1) There are cases where people may be voiced but are not playing, not every channel is ##werewolf after all and we should not impose that a channel MUST be run a particular way when voicing users in order to allow the bot to work.
2) Voice is not necessarily indicated by the + character, so if you wanted to check this for real you'd actually need to hit up ISUPPORT to check which character corresponds to +v.